sstables/sstables.cc

```diff
@@ -1408,20 +1408,25 @@ size_t sstable::total_reclaimable_memory_size() const {
 }
 
 size_t sstable::reclaim_memory_from_components() {
-    size_t total_memory_reclaimed = 0;
+    size_t memory_reclaimed = 0;
```

Comment:
Please add some motivation to the patches. It is ok for it to be short and just say that: in the next patch we want to use this to decide which component to reload, so keep the accounting in a member variable.

Reply:
Done.